src/components/Summary.tsx

```diff
@@ -102,6 +122,31 @@ export default class Summary extends React.Component<SummaryProps, SummaryState>
             </Select>
         </FormControl>
 
+
+        let notifyPractitionersSelector = null;
+        if (this.state.practitioners) {
+            let currentSelection = this.state.practitioners.filter(
+                (p: IPractitioner) => {
+                    return patient.generalPractitioner?.find((gpRef: IReference) => {
+                        return gpRef.type === "Practitioner" && p.id === gpRef.id;
+                    });
+                });
+            notifyPractitionersSelector = <Autocomplete
+                multiple
+                size="small"
+                defaultValue={currentSelection}
+                options={this.state.practitioners}
+                getOptionLabel={(option) => this.getPractitionerLabel(option as IPractitioner)}
+                renderInput={(params) => <TextField {...params} placeholder={"Practitioners"}/>}
+                onChange={(event: any, value: (string | IPractitioner)[]) => {
+                    patient.generalPractitioner = value.map((v) => ({
+                        type: "Practitioner",
+                        id: (v as IPractitioner).id
```

Comment:
@hannahburkhardt I think the generalPractioner field is a [Reference ](https://www.hl7.org/fhir/references.html#Reference)element.  Therefore I think the format here should be as such 
`{
         "type": "Practitioner",
          "reference": "Practitioner/[id]"
        }`  in order for the FHIR search by general practitioner, e.g  `/Patient?general-practitioner=[id]`, to work. 

```suggestion
                        reference: `Practitioner/${(v as IPractitioner).id}`
```

Reply:
Great catch, thank you Amy!